### DISCUSSION TITLE
Get dirty value from formikRef

### DISCUSSION THREAD

**Discussion by JoeVanGundy** [jaredpalmer/formik#3545] [Q&A]
I need to be able to handle submitting and checking the `dirty` value from outside of the `<Formik>` component (the submit button is in a modal footer).

To do this, I use `const formikRef = useRef(null)`.

and I pass that as `<Formik innerRef={formikRef} ...`

I'm able to get the submit button to work, but I can't seem to get the correct`dirty` value.

I saw [this discussion](https://github.com/jaredpalmer/formik/discussions/3310), which mentioned this may be available in v3 of formik. I pulled down `^3.0.0-next.8`, but I don't see `useFormikSubscription` anywhere.

Is there any of doing this anywhere?
Also if that has just been renamed, is there a place for the documentation on v3?

**Comment by johnrom** [comment]
duplicate of https://github.com/jaredpalmer/formik/discussions/3310

answered by https://github.com/jaredpalmer/formik/discussions/3310#discussioncomment-2562459
